### DISCUSSION TITLE
How to use SCPI commands to load saved settings and calibrations directly from my computer？

### DISCUSSION THREAD

**Discussion by Michaelli333** [jankae/LibreVNA#214] [General]
Hi Jan，How to use SCPI commands to load saved settings and calibrations directly from my computer？There are only 'query' commands in the Programming Guide for Setup Load or Calibration Load. Are there any SCPI commands that I can load local documents in my computer?
Best regards Li

**Comment by jankae** [comment]
Hi Li,
you can load calibration and setup files with the 'query' commands. My thoughts were that you are sort of asking whether it can load the file specified and that is why I chose queries. Maybe normal commands would have been the more intuitive choice...
In any case, you can use them to load these files. If you are still having trouble with it, please show me your exact command and the response to it.
Best regards
Jan

**Reply by Michaelli333** [reply]
Thank you for your prompt response, I'm going to give it a try.
